collector/src/compile/benchmark/mod.rs

```diff
@@ -458,6 +507,42 @@ impl Benchmark {
 
         Ok(())
     }
+
+    fn create_test_cases(
+        &self,
+        scenario: &Scenario,
+        profile: &Profile,
+        backend: &CodegenBackend,
+        target: &Target,
+    ) -> Vec<CompileTestCase> {
+        self.patches
```

Comment:
Are we expecting this to get more complicated later? It seems a bit wasteful (I guess fine in practice...) to produce a relatively large vec and then dedup most of it away.

Reply:
Removed the vec/set dedup and sorting.